sway/commands/move.c

```diff
@@ -124,6 +124,10 @@ static struct cmd_results *cmd_move_container(struct sway_container *current,
 					return cmd_results_new(CMD_INVALID, "move",
 							expected_syntax);
 				}
+				if (argv[3][0] < '0' || argv[3][0] > '9') {
+					return cmd_results_new(CMD_INVALID, "move",
+							"Invalid workspace number '%s'", argv[3]);
```

Comment:
Can we not use workspace 10? 11? etc?

Reply:
This just checks that the first character is a number, and if it isn't, bail. If it is, the logic is handled by `get_workspace_by_number`.

Reply:
Oh, duh. Can you use `isdigit` from `ctype.h` instead?